conda/petsc/build.sh

```diff
@@ -16,7 +16,7 @@ export CXXFLAGS=$(echo ${CXXFLAGS:-} | sed -E 's@\-fdebug\-prefix\-map[^ ]*@@g')
 export FFLAGS=$(echo ${FFLAGS:-} | sed -E 's@\-fdebug\-prefix\-map[^ ]*@@g')
 export FCFLAGS="$FFLAGS"
 export HYDRA_LAUNCHER=fork
-export LIBS="-lmpifort -lgfortran"
+#export LIBS="-lmpifort -lgfortran"
```

Comment:
@cticenhour @milljm 


Could you remind me why we need "LIBS="-lmpifort -lgfortran"" ? PETSc should figure that out automatically

Reply:
This line was added by @milljm when we were working on ARM Mac support. This may not be necessary anymore (or may only be necessary for Darwin ARM64).

Reply:
I was asking because it broke hypre builds

Do we have  Darwin ARM64 target anywhere?

Reply:
After remove "LIBS="-lmpifort -lgfortran"",  we can build conda packages correctly on both Mac and Linux (stack)

Reply:
That target is being created now. If you need to remove this, I will test it on an M1 machine to see if it is still necessary.

Reply:
Yes, I like to remove it. Or at least we should do something like this. I am not saying the syntax is correct. But you know what I meant 

```
#IF DARMIN_ARM64

#ENDIF
```

Reply:
Go ahead and make the change. Putting it in an IF is my plan if I do find that it's needed on ARM64.

Reply:
@fdkong remove this please

Reply:
Thanks. I will remove extra flags then